### PR TITLE
Thunder me bolt

### PR DESCRIPTION
**Pull request by wiktormowinski**

### CODE REVIEW

File: dasharo-compatibility/me-dock-comp.robot [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+...                     AND
+...                     Skip If    not ${DASHARO_INTEL_ME_MENU_SUPPORT}    Dasharo Intel ME menu not supported
+...                     AND
+...                     Set UEFI Option    MeMode    Enabled
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+TMD004.001 USB Type-C Display output With Me Disabled(semi-automatic)
```
Comment (mkopec) [inline dasharo-compatibility/me-dock-comp.robot:30]: Any chance to put these in the same file as the existing USB-C / thunderbolt tests? Maybe as a variant with suffix `(ME disabled)`

ideally the regular and ME disabled cases would prepare the ME state call a common keyword to do the test

Comment (wiktormowinski) [reply dasharo-compatibility/me-dock-comp.robot]: ok how about that : https://github.com/Dasharo/open-source-firmware-validation/pull/488/commits/eeda739962e40f269941ae962e34ddb0110c76b1

Comment (mkopec) [reply dasharo-compatibility/me-dock-comp.robot]: That's better, thanks.

**Review by mkopec — APPROVED** [review]

**Comment by mkopec** [comment]
@wiktormowinski could you rebase this PR? It does not rebase cleanly

**Comment by mkopec** [comment]
now precommit is unhappy :\

**Comment by wiktormowinski** [comment]
yeah, classic pre-commit always makes a scene

**Comment by mkopec** [comment]
@wiktormowinski you can just run `pre-commit run --all-files` on your local machine, it'll apply fixes automatically for you

**Comment by wiktormowinski** [comment]
neat

**Review by mkopec — APPROVED** [review]

**Comment by mkopec** [comment]
@wiktormowinski I'm getting an error when merging:

```
Total 0 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
remote: error: GH006: Protected branch update failed for refs/heads/develop.
remote:
remote: - This branch must not contain merge commits.
remote:   Found 1 violation:
remote:
```

Please rebase on develop - you *merged* instead of rebasing

**Pull request merged**